protocol/x/clob/abci.go

```diff
@@ -104,6 +104,12 @@ func EndBlocker(
 
 	// Prune any rate limiting information that is no longer relevant.
 	keeper.PruneRateLimits(ctx)
+
+	telemetry.ModuleSetGauge(
+		types.ModuleName,
+		metrics.GetMetricValueFromBigInt(keeper.GetInsuranceFundBalance(ctx)),
+		metrics.InsuranceFundBalance,
+	)
```

Comment:
nit: could put in a different function or have a clear comment that this is for any metrics that you want to emit every block